test/functionalTest/cases/1492_limit_zero/limit_zero_entities.test

```diff
@@ -0,0 +1,142 @@
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+limit_zero_entities
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. GET /v2/entities with 'limit' == 0 and options=count and see no entities and fiware-total-count: 0
+# 02. POST /v2/entities to create E1 entity
+# 03. POST /v2/entities to create E1 entity
+# 04. GET /v2/entities with 'limit' == 0 and options=count and see no entities and fiware-total-count: 2
+# 05. GET /v2/entities with 'limit' == 0 and see no entities
+#
+
+echo "01. GET /v2/entities with 'limit' == 0 and options=count and see no entities and fiware-total-count: 0"
+echo "======================================================================================================"
+orionCurl --url "/v2/entities?limit=0&options=count"
+echo
+echo
+
+
+echo "02. POST /v2/entities to create E1 entity"
+echo "========================================="
+payload='{
+  "id": "E1",
+  "type": "T"
+}'
+orionCurl --url /v2/entities -X POST --payload "$payload"
+echo
+echo
+
+
+echo "03. POST /v2/entities to create E2 entity"
+echo "========================================="
+payload='{
+  "id": "E2",
+  "type": "T"
+}'
+orionCurl --url /v2/entities -X POST --payload "$payload"
+echo
+echo
+
+
+echo "04. GET /v2/entities with 'limit' == 0 and options=count and see no entities and fiware-total-count: 2"
+echo "======================================================================================================"
+orionCurl --url "/v2/entities?limit=0&options=count"
+echo
+echo
+
+
+echo "05. GET /v2/entities with 'limit' == 0 and see no entities"
+echo "=========================================================="
+orionCurl --url "/v2/entities?limit=0"
+echo
+echo
+
+
+
+--REGEXPECT--
+01. GET /v2/entities with 'limit' == 0 and options=count and see no entities and fiware-total-count: 0
+======================================================================================================
+HTTP/1.1 200 OK
+Content-Length: 2
+Content-Type: application/json
+Fiware-Total-Count: 0
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+[]
+
+
+02. POST /v2/entities to create E1 entity
+=========================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/entities/E1?type=T
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+03. POST /v2/entities to create E2 entity
+=========================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/entities/E2?type=T
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+04. GET /v2/entities with 'limit' == 0 and options=count and see no entities and fiware-total-count: 2
+======================================================================================================
+HTTP/1.1 200 OK
+Content-Length: 2
+Content-Type: application/json
+Fiware-Total-Count: 0
```

Comment:
As I said at https://github.com/telefonicaid/fiware-orion/pull/3994/files#r783052948:

```suggestion
Fiware-Total-Count: 2
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3994/commits/ff2b2251b1f0a0a1703c4e40b51d163f72bcede8